src/pages/style.css

```diff
@@ -216,11 +216,8 @@ and negative margin.
 */
 .g-offset-scroll-margin-top {
   --permalink-scroll-offset: 24px;
-  --full-header-height: calc(
-    var(--navigation-header-height) + var(--alert-bar-height)
-  );
   --total-offset: calc(
-    var(--full-header-height) + var(--permalink-scroll-offset)
+    var(--sticky-bars-height) + var(--permalink-scroll-offset)
```

Comment:
🎨 I think we probably don't need `--sticky-bars-height` anymore since it's just the value of another variable, but we can resolve that in another PR!

Reply:
Sounds good, great call 💯  I'll resolve as part of 🎟️ [[sidebar slash top nav] fix mismatch in CSS height of top bars vs reality, especially on mobile](https://app.asana.com/0/1202114367927919/1202316169578550/f)